oxenss/storage/database.cpp

```diff
@@ -287,6 +287,30 @@ ALTER TABLE messages ADD COLUMN namespace INTEGER NOT NULL DEFAULT 0;
             )");
         }
 
+        if (!db.tableExists("revoked_subkeys")) {
+            log::info(logcat, "Upgrading database schema");
+            db.exec(R"(
+CREATE TABLE revoked_subkeys (
+    id INTEGER,
+    owner INTEGER REFERENCES owners(id) ON DELETE CASCADE,
+    subkey TEXT NOT NULL,
+    timestamp INTEGER NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    UNIQUE(subkey)
+);
+
+CREATE TRIGGER IF NOT EXISTS subkey_autoclean
+    AFTER INSERT ON revoked_subkeys WHEN (select count(*) FROMrevoked_subkeys WHERE owner = NEW.owner) > 50
```

Comment:
Missing space in the WHEN query.  Also style consistency:  `select count(*)` -> `SELECT COUNT(*)`